### PR TITLE
Fix hitcircle selection area being too large

### PR DESCRIPTION
**Pull request by smoogipoo**

### CODE REVIEW

File: osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs [hunk 1]
```diff
@@ -172,7 +170,7 @@ protected override void UpdateStateTransforms(ArmedState state)
 
         public Drawable ProxiedLayer => ApproachCircle;
 
-        private class HitArea : Drawable, IKeyBindingHandler<OsuAction>
+        public class HitReceptor : Drawable, IKeyBindingHandler<OsuAction>
```
Comment (peppy) [inline osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs]: why did you rename the class but not the variable?

Comment (smoogipoo) [reply osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs]: The rename prevents this:
```
public OsuAction? HitAction => HitArea.HitAction; // Compiler error (what is `HitArea`?)
public HitArea HitArea;
```

Open to suggestions.

Comment (peppy) [reply osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs]: right... i guess it's ok for now then.
